services/login-sync/bin/arvados-login-sync

```diff
@@ -57,18 +57,21 @@ begin
 
   pwnam = Hash.new()
   logins.reject! do |l|
-    return false if pwnam[l[:username]]
-    begin
-      pwnam[l[:username]] = Etc.getpwnam(l[:username])
-    rescue
-      if skip_missing_users
-        STDERR.puts "Account #{l[:username]} not found. Skipping"
-        true
-      end
+    if pwnam[l[:username]]
+      false
     else
```

Comment:
Actually this can be simplified further to `if not pwnam[l[:username]]` and pull up the else clause.  The works because if statements without an "else" clause evaluate to nil, which means the login is not rejected (it is accepted.)

Reply:
Done